app/models/devise_token_auth/concerns/user.rb

```diff
@@ -44,6 +44,28 @@ def email_required?; false; end
     def email_changed?; false; end
     def will_save_change_to_email?; false; end
 
+    if Devise.rails51?
+      def postpone_email_change?
```

Comment:
> Similar blocks of code found in 2 locations. Consider refactoring.
Humm, I think keeping code like this helps to make it clear this method overrides Devise method that.
Which is better, keeping or refactoring?

ref. https://github.com/plataformatec/devise/blob/f48b6f1651233a39c52dd70e9ac6c35f70086062/lib/devise/models/confirmable.rb#L279-L299

Reply:
I would prefer moving this to a concern/module, this file has too much code and we should try to clean this lib as much as possible

Reply:
Thank you for reply, @MaicolBen !
I fully agree to try to clean this library!
I moved `postpone_email_change?` method to module, so please take a look again :smile_cat:

Reply:
thanks!